tools/git-sync-deps

```diff
@@ -244,7 +244,7 @@ def main(argv):
 
   git_sync_deps(deps_file_path, argv, verbose)
   subprocess.check_call(
-      [sys.executable,
+      ['python',
```

Comment:
Do you think we could get around changing this file in some way? I'd love to keep as many upstream files unchanged as possible to avoid merge conflicts with the changes from upstream in the future.

It's not a blocker to me, I just want to find out if there are other ways :)

Reply:
I don't like this change very much, but it seems necessary. Inside cygwin, sys.executable returns an invalid path